src/main/savestates.c

```diff
@@ -476,6 +477,7 @@ static int savestates_load_m64p(struct device* dev, char *filepath)
 
     *r4300_cp0_next_interrupt(&dev->r4300.cp0) = GETDATA(curr, uint32_t);
     dev->vi.next_vi = GETDATA(curr, uint32_t);
+    if (dev->vi.next_vi <= cp0_regs[CP0_COUNT_REG]) dev->vi.next_vi += UINT64_C(0x100000000);
```

Comment:
this won't be necessary if https://github.com/mupen64plus/mupen64plus-core/pull/702 is merged